Control/test/public/page-about-mocha.js

```diff
@@ -57,7 +57,7 @@ describe('`pageAbout` test-suite', async () => {
   });
 
   it('should request info about Kafka and store in frameworkInfo.statuses as RemoteData', async () => {
-    const kafka = await page.evaluate(() => window.model.frameworkInfo.statuses.kafka);
+    const kafka = await page.evaluate(() => window.model.frameworkInfo.statuses.notification);
```

Comment:
Could you please make sure all `kafka` naming occurrences are updated to avoid future confusion?
```suggestion
    const notification = await page.evaluate(() => window.model.frameworkInfo.statuses.notification);
```